learn/tooling-guide/visual-studio-code-extension/language-support/language-support.md

```diff
@@ -2,16 +2,127 @@
 layout: ballerina-tooling-guide-left-nav-pages-swanlake
 title: Language Support
 permalink: /learn/tooling-guide/visual-studio-code-extension/language-support/
-active: language-support
-intro: The sections below describes the language support features of the Ballerina VS Code extension.  
+active: intellisense
+intro: The Visual Studio Code Ballerina extension brings in language support to enhance the development experience and increase its efficiency. This is built into the extension via a Language Server implementation, which consists of the below features.
+description: The Visual Studio Code Ballerina extension brings in language support to enhance the development experience and increase its efficiency. This is built into the extension via a Language Server implementation, which consists of the below features.
 redirect_from:
-  - /learn/tooling-guide/vs-code-extension/language-support
-  - /learn/tooling-guide/vs-code-extension/language-support/
+  - /learn/tools-ides/vscode-plugin/language-intelligence
+  - /learn/tools-ides/vscode-plugin/language-intelligence/
+  - /learn/vscode-plugin/language-intelligence/
+  - /learn/vscode-plugin/language-intelligence
+  - /learn/tools-ides/setting-up-visual-studio-code/language-intelligence
+  - /learn/tools-ides/setting-up-visual-studio-code/language-intelligence/
+  - /learn/setting-up-visual-studio-code/language-intelligence
+  - /learn/setting-up-visual-studio-code/language-intelligence/
+  - /learn/language-intelligence
+  - /learn/language-intelligence/
+  - /swan-lake/learn/getting-started/setting-up-visual-studio-code/language-intelligence/
+  - /swan-lake/learn/getting-started/setting-up-visual-studio-code/language-intelligence
+  - /learn/tooling-guide/vs-code-extension/language-support/language-intelligence
+  - /learn/tooling-guide/vs-code-extension/language-support/language-intelligence/
+  - /learn/tooling-guide/visual-studio-code-extension/language-support/language-intelligence
+  - /learn/tooling-guide/visual-studio-code-extension/language-support/language-intelligence/
   - /learn/tooling-guide/visual-studio-code-extension/language-support
 ---
 
-## Language Intelligence
+## IntelliSense
+
+The below are the features related to IntelliSense.
```

Comment:
Do we need this line?

Reply:
yes, better to have a description for the title.

Reply:
Removed.